scripts/e2e-deploy.sh

```diff
@@ -98,6 +98,7 @@ function check_var() {
 }
 
 check_var "REGISTRY_REDHAT_IO_SECRETS"
+check_var "QUAY_IO_SECRETS"
```

Comment:
This is lining up our deploy script with recent changes to the e2e deploy process.